model/infinispan/src/main/java/org/keycloak/keys/infinispan/InfinispanPublicKeyStorageProvider.java

```diff
@@ -152,15 +233,7 @@ public KeyWrapper getPublicKey(String modelKey, String kid, String algorithm, Pu
             }
 
```

Comment:
Nitpick: Since loading of keys is more flexible now (also with introducing the maxCacheTime etc), does it makes sense to add some small debug message always when keys are reloaded (either manually or automatically?). Maybe somewhere around the line 230 of your PR just introduce logger.debugf somewhere like:
```
            ...
            if (existing == null) {
                logger.debugf("Reloading keys for model key '%s'.", modelKey);
                task.run();
            } ....
```

Reply:
Done!